README.md

```diff
@@ -98,11 +102,19 @@ The above method adds "setWindspeed" as a new javascript function that the
 blockly generated code can use.  In this example, it modifies a parameter in the
 simulation store.
 
-If you want to set a default block program modify the contents of `src/assets/blockly-authoring/code/basic-setup.xml`
-or `src/assets/blockly-authoring/code/nested-loops.xml`.
-You can copy your current workspace (from eg http://localhost:8080/) by viewing the
-developer tools, clicking on the "Application" tab, and looking in Local Storage. The
-XML will be stored in local storage under the key "blockly-workspace".
+##### Adding the block to a toolbox
+For a new block to be available in the list of blocks that a user can add from the Blocks panel, it must be added to one or both of the toolboxes located in `src/assets/blockly-authoring/toolbox/first-toolbox.xml` and `src/assets/blockly-authoring/toolbox/full-toolbox.xml`. Modify one or both of these XML files to include your new block. Create a new XML `block` element nested inside the proper `category` element. Be sure to set the `type` to the unique camelCase key that was created for your block (e.g., `myExampleBlock`). For example, the add volcano block is added to the volcano category by inserting `<block type="addVolcano"></block>` inside of the volcano category element. If your block requires default fields, then add the appropriate `field` elements inside of the `block` element.
+
+##### Setting the default block program
+To change the default block program that is shown in the Blocks panel when the app is loaded, modify the contents of `src/assets/blockly-authoring/code/basic-setup.xml`
+or `src/assets/blockly-authoring/code/nested-loops.xml`. Depending on your settings, the program specified in one of these XML files will be loaded as the default program.
+
+You can also create a program in the Blocks panel (from eg http://localhost:8080/), access a copy of the XML defining the program, and copy it into one of the above files. To access the XML of the current program in the Blocks panel by doing the following:
+* in GeoCode open the model options and press Save current code to local storage
```

Comment:
⬆ Does this `Save current code to local storage` exist on recent branches?

Reply:
It's on master as of now.  I'll keep my eye on it and tweak this doc as needed.